Problem3/ip_repo/p3_2_sort_1.0/drivers/p3_2_sort_v1_0/src/p3_2_sort.c

```diff
@@ -0,0 +1,26 @@
+
+
+/***************************** Include Files *******************************/
+#include "p3_2_sort.h"
+
+/* My thought on the driver code, feel free to remove these comments once done -- by yuyuranium */
+/* void sort(unsigned char *arr)
+ * {
+ *   loop i = 1 ~ 4:
+ *     assemble 4 elements in arr into a u32
+ *     write the u32 to the memory-mapped register[i] via AXI 
+ *   write 1 to reg0
+ *
+ *   while (reg0 == 1):
+ *     read reg0
+ *
+ *   loop i = 1 ~ 4:
+ *     retrieve u32 data via AXI and store to temp
+ *     loop j = 1 ~ 4:
+ *       arr[i * 4 + j] = temp & 0xff
+ *       temp = temp >> 8
+ * }
+ *
+ */

```

Comment:
I added my thoughts (some pseudo codes) on the driver codes. @WillyChennnn maybe you want to take a look.

Reply:
Ok, I will complete my code after branch merged